webpack.common.js

```diff
@@ -13,12 +13,12 @@ module.exports = {
   },
   module: {
     rules: [
-      {
-        test: /\.(js|jsx)$/,
-        enforce: 'pre',
-        exclude: /node_modules/,
-        loader: 'eslint-loader',
-      },
+      // {
```

Comment:
If we won't use these rules, I'm ok removing them entirely instead of commenting them out.

Reply:
These lines of code make `eslint` run as a step of the `webpack` compilation. This was temporary disabled, so if we merge this to `master` as it is, compilation won't work (because of our 300+ errors), so temporary disabling the linting step will give us time to fix our errors.